### ISSUE TITLE
Enable cache-control in fontsource API request header

### ISSUE THREAD

**Issue by catiex** [DecliningLotus/fontsource#776]
### Describe the feature you'd like to request

I have noticed Fontsource currently doesn't enable caching font files downloaded from API requests. Is it possible to enable `Cache-Control` in the request header with [JSDelivr](https://www.jsdelivr.com/) CDN?

### Describe the solution you'd like

Users don't need to download font files from CDN every time if they choose to use the Fontsource API.

**Comment by ayuhito** [comment]
We're currently working with `jsDelivr` for a custom CDN endpoint. The current API was an [extremely quick rewrite](https://github.com/fontsource/fontsource/pull/734) from the old one (which completely stopped working when v5 was released) so we're still missing out on a lot of caching rules. I've just had not much time to work on this project these days so progress to implement all the handlers for the rewritten API properly has been slow.

I've got a branch which implements a much better Download API that follows optimal practices, which we'll then layer the jsDelivr CDN on top for edge caching.
